src/actions/guides/getting-started/installing.cr

```diff
@@ -268,6 +272,51 @@ class Guides::GettingStarted::Installing < GuideAction
 
     Should return `psql (PostgreSQL) 10.x` or higher.
 
+    ### 3. Create a database user and set a password
+
+    You will need to create a database user with proper permissions to allow Lucky to manage the application database.
+    There are several ways to accomplish this, but fortunatly PostgreSQL ships several command line tools to simplify
+    this step. You will need to become the 'postgres' user:
+    ```bash
+    sudo su - postgres
+    ```
+
+    Now create the user, and set the roles `createrole`, `superuser`, `createdb` then enter a password. In
+    this example below, we will create a user named 'lucky' and set a password of 'lucky'.
+    ```bash
+    createuser -dsrP lucky
+    ```
+
+    > Due to a bug in Lucky version 1.20, you will have to create a database with the same name as your
```

Comment:
Technically not a bug, and it was introduced in 1.1.0. This was intentional, it was just a poor choice :rofl: 

I think the wording is fine, but the version should say `1.2.0` at least

Reply:
OK, I will make some wording adjustments and fix the version string.